client/src/routes/profile.js

```diff
@@ -161,6 +146,26 @@ class Profile extends Component {
     }
   }
 
+  static getDerivedStateFromProps (props, state) {
+    if (props.uuid !== state.uuid) {
+      return {
+        uuid: props.uuid,
```

Comment:
It looks like we're just syncing `state.uuid` for every change in `props.uuid`, which is [an antipattern](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html).

Reply:
Isn't our code very similar [to the documentation](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#alternative-1-reset-uncontrolled-component-with-an-id-prop)? The only difference is the variable naming. 

In the future, we might want to refactor this with `key={props.uuid}` as recommend though?

Reply:
Oh true, my bad.